pages/1.13/overview/concepts/index.md

```diff
@@ -242,7 +242,7 @@ A DC/OS component is a DC/OS system service that is distributed with DC/OS.
 
 Most components run on the master nodes, but some (likr mesos-agent) run on the agent nodes.
 
-**Examples:** Mesos, Marathon, Mesos-DNS, Bouncer, Admin Router, DC/OS Package Manager (Cosmos), History Service, etc.
+**Examples:** Mesos, Marathon, Mesos-DNS, Bouncer, Admin Router, DC/OS Package Manager (Cosmos), History Service (deprecated), etc.
```

Comment:
Again, probably don't need to add deprecated here. Maybe instead we should just remove `History Service` as one of the examples.